source/install/config-proxy-nginx.rst

```diff
@@ -0,0 +1,88 @@
+.. _config-proxy-nginx:
+
+Configuring NGINX as a proxy for Mattermost Server
+==================================================
+
+**To configure NGINX as a proxy**
+
+1. Log into the server that hosts NGINX and open a terminal window.
+2. Create a configuration file for Mattermost.
```

Comment:
That's a reasonable suggestion, but I'd prefer to keep it as I've written it. The reason is that it allows for the command to be on its own line, which makes the job of quickly scanning and copy/pasting the commands into a terminal easier to do.  It provides a greater visual separation between the instruction and the code to execute, yet still allows people to go through it more slowly if they wish and understand what's going on.